client/src/components/PosterStack/index.js

```diff
@@ -72,11 +55,15 @@ function PosterStack() {
         }
       })
 
-      if (!down && gone.size === cards.length)
+      if (!down && gone.size === movies.length)
         setTimeout(() => gone.clear() || set((i) => to(i)), 600)
     }
   )
 
+  if (movies.length === 0) {
+    return <EmptyMovie />
```

Comment:
this is fine, but I want to challenge you to think about what tradeoffs you made, if any, about managing this empty state in this Component as opposed to the parent Component.

Especially when you get to handling loading and error state